Translation to German

OromeNahar opened the pull request:
I translated the application to German

slgobinath commented on safeeyes/safeeyes/config/lang/de.json:
```diff
@@ -0,0 +1,45 @@
+        ]
+    },
+    "messages": {
+        "ready_for_a_break": "Sind Sie bereit für eine Pause in {} Sekunden",
```
Google translates it as "Are you ready for a break in {} seconds?". If this is actually in a question format could you please change it to a command/statement like "Be ready for a break in {} seconds"

slgobinath commented on safeeyes/safeeyes/config/lang/de.json:
```diff
@@ -0,0 +1,45 @@
+        "strict_break": "Strikte Pause (Überspringen nicht möglich)",
+        "audible_alert": "Akustisches Signal am Ende der Pause",
+        "language": "Sprache",
+        "enable": "Safe Eyes Aktivieren",
```
This is translated to "Safe Eyes Activate". Could you please confirm the word order in German? (I feel Enable/Activate Safe Eyes" is more sensible than "Safe Eyes enable") I don't know how it is in German and even I am not a native English speaker. Just for a clarification :-)

OromeNahar commented:
Yes I will change it later. Your suggestions pretty good.

Am 05.02.2017 14:40 schrieb "Gobinath" <notifications@github.com>:

> *@slgobinath* commented on this pull request.
>
> Hi,
> Thank you for your contribution to Safe Eyes to make it better.
> As I usually say in translation reviews, I don't know even a gingle letter
> in German. However, I double check the translations using Google Translator
> to avoid general human errors. Still Google Translator is not a perfect
> tool to rely on. If you feel that the commented translations are fine and
> do not need a review, please do let me know. If there are any corrections,
> please resolve them.
>
> Thank you
> ------------------------------
>
> In safeeyes/safeeyes/config/lang/de.json
> <https://github.com/slgobinath/SafeEyes/pull/55#pullrequestreview-20169325>
> :
>
> > +        "long_break_exercises": [
> +            "Laufen Sie ein wenig",
> +            "Lehnen Sie sich zurück und entspannen Sie sich"
> +        ],
> +        "short_break_exercises": [
> +            "Schließen Sie die Augen",
> +            "Rollen Sie mit den Augen",
> +            "Rotieren Sie mit den Augen im Uhrzeigersinn",
> +            "Rotieren Sie mit den Augen gegen den Uhrzeigersinn",
> +            "Zwinkern Sie mit den Augen",
> +            "Fokussieren Sie sich auf einen weit entfernten Punkt",
> +            "Trinken Sie ein wenig Wasser"
> +        ]
> +    },
> +    "messages": {
> +        "ready_for_a_break": "Sind Sie bereit für eine Pause in {} Sekunden",
>
> Google translates it as "Are you ready for a break in {} seconds?". If
> this is actually in a question format could you please change it to a
> command/statement like "Be ready for a break in {} seconds"
> ------------------------------
>
> In safeeyes/safeeyes/config/lang/de.json
> <https://github.com/slgobinath/SafeEyes/pull/55#pullrequestreview-20169325>
> :
>
> > +        "safe_eyes_is_disabled": "Safe Eyes ist deaktiviert",
> +        "next_break_at_noon": "Nächste Pause um {}",
> +        "next_break_at_am": "Nächste Pause um {} AM",
> +        "next_break_at_pm": "Nächste Pause um {} PM"
> +    },
> +    "ui_controls": {
> +        "skip": "Überspringen",
> +        "short_break_duration": "Kleine Pause Intervall (in Sekunden)",
> +        "long_break_duration": "Lange Pause Intervall (in Sekunden)",
> +        "interval_between_two_breaks": "Intervall zwischen zwei Pausen",
> +        "no_of_short_breaks_between_two_long_breaks": "Anzahl von kleinen Pausen zwischen zwei langen Pausen",
> +        "time_to_prepare_for_break": "Zeit zur Vorbereitung für die Pause (in Sekunden)",
> +        "strict_break": "Strikte Pause (Überspringen nicht möglich)",
> +        "audible_alert": "Akustisches Signal am Ende der Pause",
> +        "language": "Sprache",
> +        "enable": "Safe Eyes Aktivieren",
>
> This is translated to "Safe Eyes Activate". Could you please confirm the
> word order in German? (I feel Enable/Activate Safe Eyes" is more sensible
> than "Safe Eyes enable") I don't know how it is in German and even I am not
> a native English speaker. Just for a clarification :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/slgobinath/SafeEyes/pull/55#pullrequestreview-20169325>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXKuwhbMJ5GXHqsPdjKLe8ZwvBcggM7sks5rZdFNgaJpZM4L3eAE>
> .
>

OromeNahar commented:
Hello,

It was a question format I changed it. For "Be ready for a break in {}
seconds" exist no good translation I changed it into "next break in {}
seconds".

I changed "Safe Eyes Activate" to "Safe Eyes Enable/Disable". The word
order ist correct. In German we say "here enable". Stuff like "enable here"
or  "disable Safe Eyes" does exist in the word word order verb + noun but I
prefere the other way arround. Maybe it's just myself :D.

For the point "about" exist a Translation but I wouldn't translate it
because it sounds better and the translation is not the best. What do u
think should I translate it?

Last question: Its my first or second time working with git. Do u get a
Notification when I change stuff or do I have to fill out a new pull
request?

best regards

On 5 February 2017 at 14:47, Christian Pe <christianpe96@gmail.com> wrote:

> Yes I will change it later. Your suggestions pretty good.
>
> Am 05.02.2017 14:40 schrieb "Gobinath" <notifications@github.com>:
>
>> *@slgobinath* commented on this pull request.
>>
>> Hi,
>> Thank you for your contribution to Safe Eyes to make it better.
>> As I usually say in translation reviews, I don't know even a gingle
>> letter in German. However, I double check the translations using Google
>> Translator to avoid general human errors. Still Google Translator is not a
>> perfect tool to rely on. If you feel that the commented translations are
>> fine and do not need a review, please do let me know. If there are any
>> corrections, please resolve them.
>>
>> Thank you
>> ------------------------------
>>
>> In safeeyes/safeeyes/config/lang/de.json
>> <https://github.com/slgobinath/SafeEyes/pull/55#pullrequestreview-20169325>
>> :
>>
>> > +        "long_break_exercises": [
>> +            "Laufen Sie ein wenig",
>> +            "Lehnen Sie sich zurück und entspannen Sie sich"
>> +        ],
>> +        "short_break_exercises": [
>> +            "Schließen Sie die Augen",
>> +            "Rollen Sie mit den Augen",
>> +            "Rotieren Sie mit den Augen im Uhrzeigersinn",
>> +            "Rotieren Sie mit den Augen gegen den Uhrzeigersinn",
>> +            "Zwinkern Sie mit den Augen",
>> +            "Fokussieren Sie sich auf einen weit entfernten Punkt",
>> +            "Trinken Sie ein wenig Wasser"
>> +        ]
>> +    },
>> +    "messages": {
>> +        "ready_for_a_break": "Sind Sie bereit für eine Pause in {} Sekunden",
>>
>> Google translates it as "Are you ready for a break in {} seconds?". If
>> this is actually in a question format could you please change it to a
>> command/statement like "Be ready for a break in {} seconds"
>> ------------------------------
>>
>> In safeeyes/safeeyes/config/lang/de.json
>> <https://github.com/slgobinath/SafeEyes/pull/55#pullrequestreview-20169325>
>> :
>>
>> > +        "safe_eyes_is_disabled": "Safe Eyes ist deaktiviert",
>> +        "next_break_at_noon": "Nächste Pause um {}",
>> +        "next_break_at_am": "Nächste Pause um {} AM",
>> +        "next_break_at_pm": "Nächste Pause um {} PM"
>> +    },
>> +    "ui_controls": {
>> +        "skip": "Überspringen",
>> +        "short_break_duration": "Kleine Pause Intervall (in Sekunden)",
>> +        "long_break_duration": "Lange Pause Intervall (in Sekunden)",
>> +        "interval_between_two_breaks": "Intervall zwischen zwei Pausen",
>> +        "no_of_short_breaks_between_two_long_breaks": "Anzahl von kleinen Pausen zwischen zwei langen Pausen",
>> +        "time_to_prepare_for_break": "Zeit zur Vorbereitung für die Pause (in Sekunden)",
>> +        "strict_break": "Strikte Pause (Überspringen nicht möglich)",
>> +        "audible_alert": "Akustisches Signal am Ende der Pause",
>> +        "language": "Sprache",
>> +        "enable": "Safe Eyes Aktivieren",
>>
>> This is translated to "Safe Eyes Activate". Could you please confirm the
>> word order in German? (I feel Enable/Activate Safe Eyes" is more sensible
>> than "Safe Eyes enable") I don't know how it is in German and even I am not
>> a native English speaker. Just for a clarification :-)
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/slgobinath/SafeEyes/pull/55#pullrequestreview-20169325>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AXKuwhbMJ5GXHqsPdjKLe8ZwvBcggM7sks5rZdFNgaJpZM4L3eAE>
>> .
>>
>


-- 
Kay-Christian Pedersen

Phone: +1 647 686 8217

slgobinath commented:
Hi @OromeNahar 
Thank you for your contribution. I know the word order depends on languages. So we need to follow whatever practiced in German.
Leaving about also fine if it makes sense.
For your last question, yes I do. In Git once you've sent a PR, the next commits will be automatically added to that PR until it is merged. I am merging your PR.

Once again, thank you for making Safe Eyes a better application :-)

Regards,
Gobinath
